src/MultiRegion/multi_region_field.jl

```diff
@@ -59,8 +59,12 @@ Base.size(f::GriddedMultiRegionField) = size(getregion(f.grid, 1))
                       getregion(f.status, r),
                       getregion(f.boundary_buffers, r))
 
-@inline reconstruct_global_field(f::AbstractField) = f
+"""
+    reconstruct_global_field(mrf)
 
+this function reconstructs the global field from a `MultiRegionField`. The global field 
+is always reconstructed on the `CPU`
```

Comment:
```suggestion
Reconstruct a global field from `mrf::MultiRegionField` on the `CPU`.
```

This might help: https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation